### ISSUE TITLE
[Token Interest] Change the page the visitors get after filling the form

### ISSUE THREAD

**Issue by lakikentaki** [MARKETProtocol/website#234]
### Description

[comment]: # (Feature or Bug? i.e Type: Bug)
*Type*: Bug

### Current Behavior
[comment]: # (Describe what actually happened.)

After filling out the interest form the users are being taken to a page that looks like this: 
![subscirbe](https://user-images.githubusercontent.com/39289842/44220964-ad78f180-a145-11e8-803d-250ae8a80815.PNG)

### Expected Behavior
[comment]: # (Describe what you expected to happen.)

After filling out the interest for the users should be taken to a page that looks something like this: 
https://thankyou.marketprotocol.io/ 

### Reproduction
[comment]: # (Describe how we can replicate the bug step by step.)

- [ ] Create a new Thank you page for people that fill out the token interest form
- [ ] Connect it to the main domain ( i.e. thankyoutoken.marketprotocol.io)
- [ ] Insert the domain link to the MailChimp form builder to replace the current form

### Definition of Done
[comment]: # (Any other information that would be useful, bullets are helpful.)

The issue is considered done once the subscribers are taken to a screen similar to our test Thank You page with all project associated resources like Whitepaper, ELI5 etc. 

### Additional Information
[comment]: # (Any other information that would be useful, content, screenshots, etc.)

This page can be created in the instapage environment and then have the domain link assigned. Let me know if you want me to do it that way.

**Comment by lakikentaki** [comment]
I have built a page, and now I just need someone to build a DNS for it as suggested so that I can copy that link into our MailChimp. Both desktop and mobile display review are desirable.  Current link: http://thankyoutoken.pagedemo.co/

**Comment by pelsasser** [comment]
closing in favor of permanent solution in #235
